src/components/Common/CaseStudyCards/CaseStudyBlocks.js

```diff
@@ -183,7 +183,61 @@ const TextAndResizedImageBlock = ({ data: { text, image } }) => (
   </Fragment>
 )
 
+const BlockRow = styled(Row)`
+  flex-direction: ${({ columnReverse }) =>
+    `column${columnReverse ? '-reverse' : null}`};
+  align-items: ${({ alignCenter }) => (alignCenter ? 'center' : null)};
+  padding-top: ${({ theme, mobile }) =>
+    mobile && mobile.top ? theme.space[mobile.top] : null};
+  padding-bottom: ${({ theme, mobile }) =>
+    mobile && mobile.bottom ? theme.space[mobile.bottom] : null};
+  justify-content: ${({ spaced }) => spaced && 'space-evenly'};
+
+  ${breakpoint('phone')`
+    justify-content: ${({ spaced }) => spaced && 'space-between'};
+  `}
+
+${breakpoint('smallTablet')`
+  flex-direction: ${({ rowReverse }) => (rowReverse ? 'row-reverse' : 'row')};
+  padding-top: ${({ theme, smallTablet }) =>
+    smallTablet && smallTablet.top ? theme.space[smallTablet.top] : null};
+  padding-bottom: ${({ theme, smallTablet }) =>
+    smallTablet && smallTablet.bottom ? theme.space[smallTablet.bottom] : null};
+`}
+
+${breakpoint('tablet')`
+  padding-top: ${({ theme, tablet }) =>
+    tablet && tablet.top ? theme.space[tablet.top] : null};
+  padding-bottom: ${({ theme, tablet }) =>
+    tablet && tablet.bottom ? theme.space[tablet.bottom] : null};
+`}
+
+${breakpoint('desktop')`
+  padding-top: ${({ theme, desktop }) =>
+    desktop && desktop.top ? theme.space[desktop.top] : null};
+  padding-bottom: ${({ theme, desktop }) =>
+    desktop && desktop.bottom ? theme.space[desktop.bottom] : null};
+`}
+`
+
+const getImage = (blockImages, index) => blockImages && blockImages[index]
+
+const normaliseAll = (arr = []) =>
```

Comment:
I think we can should use more specific names than `arr` , `genericContents` maybe?

Reply:
Using `arr` here just refers to the argument name, what we get out is a an array of generic blocks

Reply:
Yeah I understand it represents an array of something and it is an argument, I just think we should be more specific about naming. Up to you.